ansible/management-searchengine.yml

```
@@ -0,0 +1,240 @@
+# Search ngine + Search Engine Client + Elasticsearch
+
```

```suggestion
- hosts: "{{ idr_environment | default('idr') }}-database-hosts"

```

is a change I had to apply in my local deployment to make the playbook run of `test104` as it allows to load the variables from the database hosts in order to set the `database_server_url` variable